<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>leftw</class>
 <widget class="QWidget" name="leftw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>81</width>
    <height>832</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>81</width>
    <height>832</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>79</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>42</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0">
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>15</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <widget class="MyButton" name="btn_chat">
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>41</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/res/pic/Chat.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="MyButton" name="btn_contact">
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>41</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/res/pic/用户.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="widget_5" native="true">
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius:30px;
image: url(:/res/pic/天猫.png);</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <spacer name="verticalSpacer_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>16</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="0">
     <widget class="MyButton" name="btn_settings">
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>41</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/res/pic/3.1 设置.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="MyButton" name="weatherButton">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/icons/weather_icons/duoyun.png);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyButton</class>
   <extends>QPushButton</extends>
   <header>mybutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
